kotlin/internal/jvm/BUILD

```diff
@@ -33,3 +35,11 @@ bzl_library(
         "@build_bazel_rules_android//android",
     ],
 )
+
+# Jdeps plugin can be enabled or disabled using the following flag ont the command line
+# --@io_bazel_rules_kotlin//kotlin/internal/jvm:enable_jdeps
+bool_flag(
+    name = "enable_jdeps",
```

Comment:
Should the name match [java_deps](https://bazel.build/reference/command-line-reference#flag--java_deps)